_pytest/mark.py

```diff
@@ -168,9 +164,13 @@ def matchkeyword(colitem, keywordexpr):
 
 
 def pytest_configure(config):
-    import pytest
+    config._old_mark_config = MARK_GEN._config
     if config.option.strict:
-        pytest.mark._config = config
+        MARK_GEN._config = config
```

Comment:
This will only support two levels, but I think it is enough (pytest "main" calling a test which uses a "testdir")